components/insight/SRC/org/openmicroscopy/shoola/util/CommonsLangUtils.java

```diff
@@ -0,0 +1,368 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2015 University of Dundee. All rights reserved.
```

Comment:
Are the methods copied from commons-lang? If so, a different copyright header might be appropriate.